### ISSUE TITLE
Unreadable published date

### ISSUE THREAD

**Issue by T-prog3** [janeczku/calibre-web#2515]
When using the `caliBlur! Dark Theme` there's some sort of overlapping bug that makes the text in Published Date unreadable.
![published-date](https://user-images.githubusercontent.com/74016497/184448742-b2c4f5f1-899e-4add-8de1-6d7eafafda56.png)

**To Reproduce**
1. Switch over to `caliBlur! Dark Theme`
2. Select a book
3. Press `Edit Metadata`
4. Create a `Published Date`
5. Now it's unreadable.

**Comment by SmartPhoneLover** [comment]
+1, same error here.

**Comment by misuchiru03** [comment]
I noticed this same thing. It's a double-formatting of the date. Somewhere in the code, when the date is entered, the proper/expected format is YYYY-MM-DD, and it does this in black text. The unreadable part comes from white/hidden text using M/D/YYYY format.
![image](https://user-images.githubusercontent.com/22798851/197152408-46dfd498-2db2-4e98-9a2c-41e090c126bf.png)
![image](https://user-images.githubusercontent.com/22798851/197152756-70fc3ac8-5ffb-4ffd-b7d0-a46ab73a8026.png)
![image](https://user-images.githubusercontent.com/22798851/197152813-58027bd2-1eb0-4bb4-a307-68e3c8b1e368.png)

I tried looking into the code to see if I could make a correction and submit it as a PR, but I've not had much time to do so.
